dashboard/app/models/level.rb

```diff
@@ -214,6 +214,7 @@ def self.key_to_params(key)
   # be passed to the client, typically to save and load user progress
   # on that level.
   def channel_backed?
+    false if self.try(:is_project_level)
```

Comment:
doesnt false only get returned here if it's the last statement? i.e. doesnt this line end up being a no-op, and we always return the result of the next line, or am i just misunderstanding ruby